src/DetailsView/components/action-and-cancel-buttons-component.tsx

```diff
@@ -26,6 +27,7 @@ export class ActionAndCancelButtonsComponent extends React.Component<ActionAndCa
                         onClick={this.props.primaryButtonOnClick}
                         disabled={this.props.primaryButtonDisabled}
                         href={this.props.primaryButtonHref}
+                        target={isEmpty(this.props.primaryButtonHref) ? '_self' : '_blank'}
```

Comment:
I'm not sure we do we need this `target` here. Can you clarify? (I'm not saying we don't need it, I just don't get the why)

Reply:
If we have a href on the primary button, it means we are using the button as a link, which should open in a new tab. Having target=_blank takes care of that.

Reply:
In normal scenarios: `target = _blank` makes the link open the `href` in a new tab. Is that the same behavior here? I am a little confused.

Reply:
Yes, it is the same behavior here.

Reply:
So, what's the scenario where we use `'_self'`???

Reply:
'_self' is the default option, it's the same as not have a target attr.